tests/integration/sugar/test_transaction.py

```diff
@@ -415,16 +381,15 @@ async def test_reliable_submission_last_ledger_expiration(self, client):
         [
             "xrpl.transaction.sign",
             "xrpl.transaction.send_reliable_submission",
-            "xrpl.account.get_next_valid_seq_number",
-            "xrpl.ledger.get_fee",
+            "xrpl.ledger.get_latest_validated_ledger_sequence",
         ],
     )
     async def test_reliable_submission_bad_transaction(self, client):
-        WALLET.sequence = await get_next_valid_seq_number(ACCOUNT, client)
         payment_dict = {
             "account": ACCOUNT,
-            "last_ledger_sequence": WALLET.sequence + 20,
             "fee": "10",
+            "last_ledger_sequence": await get_latest_validated_ledger_sequence(client)
```

Comment:
Does autofill not fill in a `last_ledger_sequence`? I thought it did....

Reply:
It does. This specific transaction doesn't autofill - I think because it is expected to fail.